examples/go_examples.md

```diff
@@ -117,13 +117,13 @@ install:
   - go get -v github.com/codeclimate/test-reporter
   - cd $GOPATH/src/github.com/golang/dep/cmd/dep && git checkout tags/v0.4.1 && go install
   - cd $GOPATH/src/github.com/codeclimate/test-reporter && git checkout tags/v0.4.3 && go install
-  - cd - && dep ensure -v -vendor-only
+  - cd $GOPATH/src/github.com/user/project && dep ensure -v -vendor-only
```

Comment:
I understand that the underlying goal of this change, it's to convey in a generalized way that you must be on the root of the project. But I worried the someone will copy/paste this and still have trouble understanding what's the intention behind this line. Do you think is worth adding a comment?

Reply:
I assume that if the audience is writing go code and using travis for their builds, they are smart enough to understand they need to replace this.

This example (initially a copy paste from my issue #292 ) aims to show how to integrate codeclimate code coverage within a golang + travis setup. Which is not very obvious as the documentation didn't mention examples (now it does).

I will amend this and also remove dependencies management which is not needed.

Reply:
This sounds good to me.